apps/dashboard/test/models/motd_formatter/motd_formatter_markdown_erb_test.rb

```diff
@@ -1,88 +1,69 @@
 require 'test_helper'
 
 class MotdFormatterMarkdownErbTest < ActiveSupport::TestCase
-  include MotdFormatter
   test "motd-formatter-md-erb returns valid motd file when given a valid motd file" do
-    path = "#{Rails.root}/test/fixtures/files/motd_valid"
-    motd_file = MotdFile.new(path)
-    formatted_motd = MotdFormatterMarkdownErb.new(motd_file)
-    expected_file = OodAppkit.markdown.render(motd_file.content)
+    with_modified_env({ 'MOTD_FORMAT': "markdown_erb",'MOTD_PATH': "#{Rails.root}/test/fixtures/files/motd_valid" }) do
+      motd_file = MotdFile.new
+      expected_file = File.read "#{Rails.root}/test/fixtures/files/motd_valid_html"
```

Comment:
This test initially used `motd_valid` but now uses `motd_valid_html`?

What's more, this test is supposed to test an `.md.erb` file not an `.html` file.

Reply:
🤦‍♂️ I see what this is now, disregard the initial comment.

Can you use parenthesis here for this `File.read` call?